test/addons/buffer-free-callback/test.js

```diff
@@ -19,17 +19,17 @@ function check(size, alignment, offset) {
 check(64, 1, 0);
 
 // Buffers can have weird sizes.
-check(97, 1, 0);
+check(97, 1, 1);
 
 // Buffers can be unaligned
 check(64, 8, 0);
 check(64, 16, 0);
```

Comment:
If the base address of the static buffer happens to be aligned on a 16 byte boundary, won’t the calculated address be the same for the 16 and 8 alignment cases?

Reply:
Actually not because the start of the new address is
https://github.com/thangktran/node/blob/3261259bd0bba00686d179c48927c36f76fcbffa/test/addons/buffer-free-callback/binding.cc#L22